react-intro/react-intro-quiz-notes.md

```diff
@@ -0,0 +1,45 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is React?
+  It is a front end library for js used to build interfaces based on components.
+
+- What is a React component?
+  A component is a piece of the ui that contains it's own logic and appearance.
+
+- How do you mount a React app (root component) to the DOM?
+  export default???
```

Comment:
Not quite! How are we actually attaching our React app to our HTML file? Look at `App.jsx`

Reply:
is it because the function that returns the dom elements is being called?